src/components/layout/CyVerseAppBar.js

```diff
@@ -352,10 +404,7 @@ function CyverseAppBar(props) {
     const onUserMenuClose = () => {
         setAnchorEl(null);
     };
-    const handleSearchClick = (event) => {
-        router.push("/" + NavigationConstants.SEARCH);
-        toggleDrawer(false);
-    };
```

Comment:
Rendered irrelevant by the DrawerItem code -- _unless_ I've misunderstood something and there's a reason we specifically need the `router.push` first here, vs. the other ones that all have `toggleDrawer` first.